main.go

```diff
@@ -58,5 +58,15 @@ func main() {
 	l.Printf("Inventory Service, version %s starting up",
 		CreateVersionString())
 
+	db, err := NewDataStoreMongo(config.Config.GetString(SettingDb))
+	if err != nil {
+		l.Fatal("failed to connect to db")
+	}
+
+	err = db.Migrate(Version, nil)
```

Comment:
@maciejmrowiec you're right about a separate db version though. I took a shortcut and reused the api version, that's confusing.